pom.xml

```diff
@@ -0,0 +1,394 @@
+                        <filters>
+                            <filter>
+                                <artifact>*:*</artifact>
+                                <excludes>
+                                    <exclude>META-INF/*.SF</exclude>
+                                    <exclude>META-INF/*.DSA</exclude>
+                                    <exclude>META-INF/*.RSA</exclude>
+                                </excludes>
+                            </filter>
+                        </filters>
+                    </configuration>
+                    <executions>
+                        <execution>
+                            <phase>package</phase>
+                            <goals>
+                                <goal>shade</goal>
+                            </goals>
+                            <configuration>
+                                <transformers>
+                                    <transformer
+                                            implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                                        <mainClass>${mainClass}</mainClass>
+                                    </transformer>
+                                </transformers>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+
+                <plugin>
+                    <groupId>org.scalatest</groupId>
+                    <artifactId>scalatest-maven-plugin</artifactId>
+                    <version>${maven-scalatest-plugin.version}</version>
+                    <configuration>
+                        <junitxml>.</junitxml>
+                        <debugForkedProcess>${maven.scalatest.debug}</debugForkedProcess>
+                        <filereports>TestSuite.txt</filereports>
+                        <testFailureIgnore>false</testFailureIgnore>
+                        <wildcardSuites>${testClasses}</wildcardSuites>
+                    </configuration>
+                    <executions>
+                        <execution>
+                            <id>test</id>
+                            <phase>test</phase>
+                            <goals>
+                                <goal>test</goal>
+                            </goals>
+                            <configuration>
+                                <reportsDirectory>${project.build.directory}/scalatest/reports</reportsDirectory>
+                                <htmlreporters>${project.build.directory}/scalatest/html</htmlreporters>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-jar-plugin</artifactId>
+                    <version>3.0.2</version>
```

Comment:
You have a pinned version and a variable for the same; both are out of sync.

Reply:
done